From 0bd47b5d57033bf7c6978f86f75b7564a6796a93 Mon Sep 17 00:00:00 2001
From: Dominik Schmidt <das1993@hotmail.com>
Date: Sun, 29 May 2016 12:24:46 +0200
Subject: [PATCH] Escape "-Wl," arguments from CMAKE_EXE_LINKER_FLAGS.

Some systems (e.g. Gentoo) pass the Linkerflags as arguments to the
gcc-suite (-Wl,-O1), which explodes with "unrecognized option '-Wl,-O1'"

This fixes issue #1494 which was introduced in 49d44595
---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index fd57f28..9532712 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -575,6 +575,7 @@ else()
         if(NOT "${CMAKE_EXE_LINKER_FLAGS}" STREQUAL "")
             separate_arguments(flags UNIX_COMMAND "${CMAKE_EXE_LINKER_FLAGS}")
             foreach(f ${flags})
+		string(REPLACE "-Wl," "" f ${f})
                 append("-L${f}" tempVar)
             endforeach()
         endif()
